pages/blog/index.vue

```diff
@@ -54,7 +54,7 @@ export default {
       const filePath = post.meta.resourcePath
       const fileName = filePath.match(/\/([^/]+)\.\w+$/)[1]
 
-      return fileName
+      return `/blog/${fileName}`
```

Comment:
Why this change?

Reply:
It's about solving a prefix issue before. It seems not needed now.